recipes/plugins_example_document.ipynb

```diff
@@ -93,8 +93,24 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.    kili.update_properties_in_assets(
</pre><p>What is this for ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/989/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
It was to add metadatatype but I deleted a part of it, fixed !